packages/playground/extensions/package.json

```diff
@@ -9,6 +9,6 @@
     "serve": "vite preview"
   },
   "dependencies": {
-    "vue": "^3.2.13"
+    "vue": "^3.2.16"
```

Comment:
Should we do this in a separate PR? So we have a clean commit?

Reply:
This is necessary for the tests to pass. 3.2.16 contains a fix for pnpm's strict dep requirements.

Reply:
Maybe we could first merge a PR for just updating vue, and then we have this switch pnpm in a clean PR